R/build-search-docs.R

```diff
@@ -314,3 +308,13 @@ is_heading <- function(node) {
 capitalise <- function(string) {
   paste0(toupper(substring(string, 1, 1)), substring(string, 2))
 }
+
+get_site_paths <- function(pkg) {
+  paths <- fs::path_rel(
+    fs::dir_ls(pkg$dst_path, glob = "*.html", recurse = TRUE),
+    pkg$dst_path
+  )
+  # do not include dev package website in search index / sitemap
+  dev_destination <- meta_development(pkg$meta)$destination
```

Comment:
using this as it might not be "dev"

Reply:
Is it always a relative path?

Reply:
yes if I understand 

https://github.com/r-lib/pkgdown/blob/23a04154b020bde5ed0cfee75d043ef5f604a649/R/package.r#L43

correctly